dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/system_jobs.jsp

```diff
@@ -0,0 +1,86 @@
+<%@page import="com.dotcms.repackage.javax.portlet.WindowState"%>
+<%@page import="com.liferay.portal.util.Constants"%>
+<%@page import="com.liferay.portal.language.LanguageUtil"%>
+<%@page import="org.quartz.CronTrigger"%>
+<%@page import="java.text.SimpleDateFormat"%>
+<%@page import="java.text.DateFormat"%>
+<%@page import="org.quartz.Trigger"%>
+<%@page import="org.quartz.JobDetail"%>
+<%@page import="java.util.Set"%>
+<%@page import="com.dotmarketing.quartz.ScheduledTask"%>
+<%@page import="com.dotmarketing.quartz.QuartzUtils"%>
+<%@page import="java.util.Date"%>
+<%@page import="java.util.ArrayList"%>
+<%@page import="java.util.Arrays"%>
+<%@page import="java.util.List"%>
+<%@page import="com.dotmarketing.business.APILocator"%>
+<%@ page import="com.dotmarketing.util.UtilMethods" %>
+<%@ page import="com.dotmarketing.beans.Host" %>
+<%@ page import="com.dotmarketing.business.web.WebAPILocator"%>
+
+<%@ include file="/html/common/uservalidation.jsp"%>
+
+<%String[] groups = QuartzUtils.getScheduler().getJobGroupNames(); %>
+<table class="listingTable">
```

Comment:
Codacy found an issue: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6213816)